scipio/src/dma_file.rs

```diff
@@ -62,10 +62,23 @@ macro_rules! bad_buffer {
     }};
 }
 
+#[inline(always)]
+/// Align value `v` up where align = 2^n.
```

Comment:
I don't like the fact that you say `align = 2^n`. It may give the reader the impression that you are to pass `n`.
I prefer that we spell this out in plain English, `align must be a power of two`